### PR TITLE
Update deprecated set-output command

### PR DESCRIPTION
**Pull request by raminqaf**
Solves #34

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ This workflow is built from multiple composite actions listed below:
 | ref               |    ❌    | The default branch of your repository | string  | ref name to checkout the repository                                                                                               |
 | publish-to-test   |    ❌    |                 true                  | boolean | If set to true, the packages are published to test.pypi.org other wise the packages are published to pypi.org                     |
 | python-version    |    ❌    |                "3.10"                 | string  | Python version for setting up poetry                                                                                              |
-| poetry-version    |    ❌    |               "1.1.12"                | string  | Poetry version to be installed                                                                                                    |
+| poetry-version    |    ❌    |               "1.2.2"                 | string  | Poetry version to be installed                                                                                                    |
```
Comment (disrupted) [inline .github/workflows/README.md:379]: why did you update Poetry? and why not to the latest version?

File: .github/workflows/python-poetry-release.yaml [hunk 2]
```diff
@@ -89,8 +88,7 @@ jobs:
           poetry-version: ${{ inputs.poetry-version }}
 
       - name: Commit and push pyproject.toml file
-        if: success()
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.13.1
```
Comment (disrupted) [inline .github/workflows/python-poetry-release.yaml:91]: afaict this action wasn't changed?

**Review by disrupted — CHANGES_REQUESTED** [review]

File: .github/workflows/python-poetry-release.yaml (hunk 2 above)
Comment (raminqaf) [reply .github/workflows/python-poetry-release.yaml:91]: No, it wasn't but I wanted to update it to the newest release (that we do after this merge).

File: .github/workflows/README.md (hunk 1 above)
Comment (raminqaf) [reply .github/workflows/README.md:379]: I can update it to 1.3.1. I was not sure if there were any breaking changes in 1.3. I was sure this is working with 1.2.x

File: actions/bump-version/action.yaml [hunk 3]
```diff
@@ -28,8 +28,8 @@ runs:
     - name: Bump version
       id: release-tag
       run: |
-        echo ::set-output name=oldTag::$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")
+        echo "oldTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> $GITHUB_OUTPUT
```
[the remark this answers is not in the document]
Comment (raminqaf) [reply actions/bump-version/action.yaml:31]: That might be true. I thought the same. We can move the Python code to a `.py` file and call it from there. WDYT?

**Comment by yannick-roeder** [comment]
What about the `java-gradle-release.yaml` workflow? It also still uses `::set-output`. I'm fine with merging this file untested since we are not using it anywhere yet and will migrate to it soon

File: .github/workflows/python-poetry-release.yaml (hunk 2 above)
Comment (disrupted) [reply .github/workflows/python-poetry-release.yaml:91]: the update is technically unnecessary 🤷🏻

File: actions/bump-version/action.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (disrupted) [reply actions/bump-version/action.yaml:31]: sounds reasonable! 👍🏻

File: .github/workflows/README.md (hunk 1 above)
Comment (disrupted) [reply .github/workflows/README.md:379]: I think it's a good idea to update Poetry, but please do so in a separate PR

**Comment by raminqaf** [comment]
@yannick-roeder I updated it even there. They are pretty basic set-output commands, I think (and hope) nothing goes wrong there!

File: scripts/python/get_version.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from configparser import ConfigParser
```
Comment (disrupted) [inline scripts/python/get_version.py:1]: is it possible to move this in the directory of the bumpversion action?

**Review by disrupted — CHANGES_REQUESTED** [review]

File: .github/workflows/java-gradle-release.yaml [hunk 5]
```diff
@@ -79,10 +79,8 @@ jobs:
       - name: Bump version
         id: evaluate-version
         run: |
-          old_version=$(grep "version=" gradle.properties | cut -d "=" -f2)
-          release_version=$(semver bump "${{ inputs.release-type }}" "${old_version}")
-          echo ::set-output "name=old-version::$old_version"
-          echo ::set-output "name=release-version::$release_version"
+          echo "old-version=$(grep "version=" gradle.properties | cut -d "=" -f2)" >> $GITHUB_OUTPUT
+          echo "release-version=$(semver bump "${{ inputs.release-type }}" "${old_version}")" >> $GITHUB_OUTPUT
```
Comment (raminqaf) [inline .github/workflows/java-gradle-release.yaml:83]: @yannick-roeder I am not really sure about the double quotes here

Comment (raminqaf) [reply .github/workflows/java-gradle-release.yaml:83]: @yannick-roeder Should shellcheck control this?

Comment (yannick-roeder) [reply .github/workflows/java-gradle-release.yaml:83]: I think the spellcheck is fine in general. If it is that complicated to change it in this MR, then you can remove it here and we will fix it with [this MR](https://github.com/bakdata/ci-templates/pull/49) where we are rewriting the output anyway

**Review by disrupted — APPROVED** [review]

**Pull request merged**